_posts/2024-06-18-trino-fest-recap.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Trino Fest 2024 recap"
+author: "Manfred Moser, Cole Bowden, Monica Miller"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2024/recap-banner.png
+---
+
+tbd
+Trino Fest 2024 is successfully in the books! While over 100 enthusiastic
+members of the community gathered in Boston, over 650 virtual attendees
+joined us worldwide to learn from our expert speakers as they discussed
+topics such as table formats, enhancements and optimizations, and use
+cases with Trino both large and small.
+<!--more-->
+
+## Re
+tbd
+
+
+## Sessions
+
+* [What’s new in Trino this summer](https://www.youtube.com/watch?v=mk3n0_tAdZY){:target="_blank"}
```

Comment:
Martin recapped everything that's happened in Trino over the last six months, taking a look at the biggest new features and how Trino development is going better than ever. He also gave a sneak peek at what we can expect soon in Trino.